pkgs/by-name/pi/pipeworld/package.nix

```diff
@@ -5,13 +5,13 @@
 
 stdenvNoCC.mkDerivation (finalPackages: {
   pname = "pipeworld";
-  version = "unstable-2023-03-02";
+  version = "unstable-2023-02-05";
```

Comment:
Isn't this a downgrade?

Reply:
It is an error, to be honest. The previous version was 2023-**02-03** not 2023-**03-02**.